### PR TITLE
Update website IA for v5

### LINKED ISSUE #3095: Revise website IA for improved findability
Reorganize content on the website according to the design proposal here: https://docs.google.com/document/d/1u6KlJ16vsSzQJvbK6YFkzhiobT6zz2FhuPufCV_zuaE/edit?usp=sharing

### PR DESCRIPTION
**Pull request by mcarrano**
This PR makes changes to the organization of pages on the website.

Closes #3095

### CODE REVIEW

**Comment by mcarrano** [comment]
@evwilkin @nicolethoen this PR should now be ready to review and merge. The remaining updates are needed in the Core and React repos and I've opened separate issues there. The only problem I'm seeing is that the Topology page does not display here, which is strange because I made to change to that except move it in the order. Any idea what's going on with that?

**Comment by nicolethoen** [comment]
the v5 branch is waiting for the topology code to be officially published from the new topology repo, so it doesn't have all the new topology demos in it yet (until that repo is completely set up - I think jeff has made good progress)

**Review by nicolethoen — APPROVED** [review]

File: packages/v4/patternfly-docs/content/design-guidelines/components/password-strength/password-strength.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: Password strength
-section: demos
+section: components
```
Comment (evwilkin) [inline packages/v4/patternfly-docs/content/design-guidelines/components/password-strength/password-strength.md:3]: Just confirming that this is the only item in this PR moving to 'components' (though it is not a standalone component)?

Comment (mcarrano) [reply packages/v4/patternfly-docs/content/design-guidelines/components/password-strength/password-strength.md:3]: It's not the only item moving, but it's the only item moving that has associated design guidelines. Therefore it's the only existing demo that requires changes in Org. Does that make sense?

Comment (evwilkin) [reply packages/v4/patternfly-docs/content/design-guidelines/components/password-strength/password-strength.md:3]: I think - just confirming "components" is the right section for this to live

**Review by evwilkin — APPROVED** [review]
LGTM

**Comment by evwilkin** [comment]
Scratch the above comment - pulling in Core/React updates also pulls in all breaking changes from those repos, which breaks the Org build and should be tackled separately from this issue.  Once those changes are pulled in & addressed the nav items should slot as expected without any additional follow-up work needed.

**Pull request merged**